documentation/src/main/asciidoc/user_guide/server_protocols_hotrod_transaction.adoc

```diff
@@ -66,12 +66,7 @@ infinispan.client.hotrod.transaction.timeout = 60000
 ====== TransactionManagerLookup Interface
 `TransactionManagerLookup` provides an entry point to fetch a {tm}.
 
-[TIP]
-====
-You can implement this interface and add your own {tm}.
-====
-
-The following implementations of `TransactionManagerLookup` are available:
+Available implementations of `TransactionManagerLookup`:
```

Comment:
As I mentioned in ISPN-9880 we should mention that you are able to implement a custom one, but this is only necessary to integrate other (i.e. AppServers) TM's
And for 99% the Generic should work for standalone or inside EAP

Reply:
@wfink ok, I thought you meant it was just confusing about the tip to implement the interface. I've updated to clarify. thx!